packages/unenv-preset/package.json

```diff
@@ -54,8 +54,8 @@
 		"wrangler": "workspace:*"
 	},
 	"peerDependencies": {
-		"unenv": "2.0.0-rc.0",
-		"workerd": "^1.20241230.0"
+		"unenv": "2.0.0-rc.1",
+		"workerd": "^1.20250124.0"
```

Comment:
I note that Miniflare is not yet updated to this version.
I guess that the process is (I keep forgetting):
- bump unenv's dep on workerd - check that it works correctly in isolation
- bump wrangler/miniflare's deps on workerd and unenv - check that they work together

E.g. via two PR